bgpd/bgp_mplsvpn.h

```diff
@@ -170,16 +170,6 @@ static inline int vpn_leak_to_vpn_active(struct bgp *bgp_vrf, afi_t afi,
 		return 0;
 	}
 
-	/* Is there an "auto" export label that isn't allocated yet? */
-	if (CHECK_FLAG(bgp_vrf->vpn_policy[afi].flags,
-		BGP_VPN_POLICY_TOVPN_LABEL_AUTO) &&
```

Comment:
Is this handled somewhere else?

Reply:
I think this is the RC of the failed tests. I added it in the bgp_mplsvpn_get_vpn_label() function